src/alasqlparser.jison

```diff
@@ -851,6 +851,32 @@ FromClause
 		{ $$ = { from: [$2], joins: $3 }; }
 */	| FROM FromTablesList JoinTablesList
 		{ $$ = { from: $2, joins: $3 }; }
+	| FROM FromTablesList JoinTablesList COMMA FromTablesList
```

Comment:
is this really the best way to do it? I feel like we are not using the parser as its meant to be used.

Reply:
OK. That is a lot of change. Lets keep this approach.